docs/ref/contrib/staticfiles.txt

```diff
@@ -293,6 +293,18 @@ content:
 
     @import url("../admin/css/base.27e20196a850.css");
 
+.. versionchanged:: 1.11
```

Comment:
These annotations should be "self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features. Also add release notes (as a possibly backwards-incompatible change due to possible infinite loops?)